integration/test_flow_integration.py

```diff
@@ -123,6 +123,38 @@ def test_write_to_v3io_stream():
     assert shard1_data == [b'1', b'3', b'5', b'7', b'9']
 
 
+def test_write_dict_to_v3io_stream():
+    stream_path = f'bigdata/test_write_dict_to_v3io_stream/{int(time.time_ns() / 1000)}/'
+    asyncio.run(SetupStream().setup(stream_path))
```

Comment:
How about cleaning after the test?
you can add stream_setup_teardown_fixture for that